act-rules-format.bs

```diff
@@ -245,183 +259,212 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
 
-Note that atomic rules in a composite rule MAY have different applicability. Because of this, not every element applicable within the composite rule is tested by every atomic rule.
+Note that input rules in a composite rule MAY have different applicability. Because of this, not every test target applicable within the composite rule is tested by every input rule.
 
 <div class=example>
-  <p><strong>Example</strong>: A composite rule about `img` elements uses results from atomic rules that have the following applicability:</p>
+  <p>**Example:** A composite rule about `img` elements uses results from rules that have the following applicability:</p>
   <ul>
-    <li><strong>Atomic rule1</strong>: All `img` element with an `alt` attribute</li>
-    <li><strong>Atomic rule1</strong>: All `img` element without an `alt` attribute</li>
+    <li>**Rule 1:** All `img` element _with_ an `alt` attribute</li>
+    <li>**Rule 2:** All `img` element _without_ an `alt` attribute</li>
   </ul>
-
-  <p>The applicability of the composite rule combines the applicability of both atomic rules. This becomes:</p>
+  <p>The applicability of the composite rule combines the applicability of both rules. This becomes:</p>
   <blockquote>
     <p>All `img` elements.</p>
   </blockquote>
 </div>
 
 
 Expectations {#expectations}
-==============================
+----------------------------
+
+An ACT Rule MUST contain one or more expectations. The expectations describe what the requirements are for [=test targets=] derived from the [applicability](#applicability). An expectation is an assertion about all [=test targets=] that is true if the [=test subject=] satisfies the [=accessibility requirement=]. When all expectations are true for a test subject, the test subject `passed` the rule. If one or more expectations are false, the test subject `failed` the rule. If there are no test targets, the [=outcome=] for the test subject is `inapplicable`.
 
-The expectations describe what the requirements are for the [=test targets=] derived from the [applicability](#applicability).
+Each expectation MUST be distinct, unambiguous, and be written in plain language. Unlike in applicability, a certain level of subjectivity is allowed in expectations. Meaning that the expectation has only one possible meaning, but that meaning MAY not fully quantifiable.
 
+Commonly, an expectation comes in the form of an "Each test target has ..." statement. In such an expectation, all test targets need to [=meet expectations|meet all expectations=] for the rule to pass. It is also possible to have an expectation where not all test targets meet all expectations. For example: "85% of all test targets have an accessible name" is an expectation that is true even when 15% of the test targets do not meet the expectations.
```

Comment:
(important) I think this paragraph needs to be rewritten, to better explain the different types of definition forms for expectations. It seems that we have two types:

1. individual test targets - the expectation applies to individual test targets, like "every test target has an accessible name"
2. sets of test targets - the expectation applies to a set of test targets, like "85% of all test targets have an accessible name"

Reply:
... building on what Shadi said, I think that the "85%..." type of expectation rules out saying anything about whether a particular test target meets the expectation. These types of rules can only say something about the set they apply to. 

(Even though an implementation could choose to report which elements count toward/from meeting the expectation. And it would probably be nice to report some calculations too, e.g. how many % had an accessible name, and how far is it from the 85% threshold).

I also don't really see why it's not always "meet **all** expectations". I agree that it's not always "all test targets", like in the 85% case, but I cannot imagine an example where we would allow for only meeting e.g. 2 out of 3 expectations in a rule. If the expectation is "85%...", this is still an expectation that needs to be met.